g11n-ws/modules/md-service-authen/src/main/java/com/vmware/vip/core/validation/ParameterValidation.java

```diff
@@ -206,7 +206,7 @@ private void validateKey(HttpServletRequest request)
 			return;
 		}
 		if (!RegExpValidatorUtils.isAscii(key)) {
-			throw new ValidationException(ValidationMsg.KEY_NOT_VALIDE);
+			throw new ValidationException(String.format(ValidationMsg.KEY_NOT_VALIDE_FORMAT, key));
```

Comment:
```suggestion
			throw new ValidationException(String.format(ValidationMsg.KEY_NOT_VALID_FORMAT, key));
```

Reply:
It is as same as your suggest in my browser.